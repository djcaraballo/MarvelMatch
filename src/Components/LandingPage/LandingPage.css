@font-face {
  font-family: 'Benton Sans', sans-serif;
  font-style:normal;
  font-weight:bold;
  src: 
      url("../../assets/BentonSans/BentonSans.otf") format("truetype")
}

@keyframes leaves {
  0% {
    transform: scale(2.0);
    -webkit-transform: scale(2.0);
  }
  100% {
    transform: scale(1.0);
    -webkit-transform: scale(1.0);
  }
}

@-webkit-keyframes leaves {
  0% {
    transform: scale(2.0);
    -webkit-transform: scale(2.0);
  }
  100% {
    transform: scale(1.0);
    -webkit-transform: scale(1.0);
  }
}

@-webkit-keyframes fadeIn {
  0% {opacity: 0;}  
  100% {opacity: 1;}
}

@-moz-keyframes fadeIn {
  0% {opacity: 0;}  
  100% {opacity: 1;}
}

@-o-keyframes fadeIn {
  0% {opacity: 0;}  
  100% {opacity: 1;}
}

@keyframes fadeIn {
  0% {opacity: 0;}  
  100% {opacity: 1;}
}

.fadeIn {
  -webkit-animation-name: fadeIn;
  -moz-animation-name: fadeIn;
  -o-animation-name: fadeIn;
  animation-name: fadeIn;
}

@keyframes fadeOut {
  0% {opacity: 1;}
  100% {opacity: 0;}
}

@-moz-keyframes fadeOut {
  0% {opacity: 1;}
  100% {opacity: 0;}
}

@-webkit-keyframes fadeOut {
  0% {opacity: 1;}
  100% {opacity: 0;}
}

.fadeOut {
  -webkit-animation-name: fadeOut;
  -moz-animation-name: fadeOut;
  -o-animation-name: fadeOut;
  animation-name: fadeOut;}

@-webkit-keyframes fadeInDown {
  0% {
    opacity: 0;
    -webkit-transform: translateY(-20px);
    transform: translateY(-20px);
  }

  100% {
    opacity: 1;
    -webkit-transform: translateY(0);
    transform: translateY(0);
  }
}

@keyframes fadeInDown {
  0% {
    opacity: 0;
    -webkit-transform: translateY(-20px);
    -ms-transform: translateY(-20px);
    transform: translateY(-20px);
  }

  100% {
    opacity: 1;
    -webkit-transform: translateY(0);
    -ms-transform: translateY(0);
    transform: translateY(0);
  }
}

.fadeInDown {
  -webkit-animation-name: fadeInDown;
  animation-name: fadeInDown;
}

.gif { 
  background: url('../../assets/comic-flip.gif');
  background-repeat: no-repeat;
  background-size: cover;
  width: 100%;  
  height: 100%;
  position: absolute; 
  top: 0; left: 0;
  -webkit-animation: fadeOut 7s forwards;
  -moz-animation: fadeOut 7s forwards;
  animation: fadeOut 7s forwards;
}

html { 
  background: #550703; 
  width:100%; 
  height: 100%; 
  overflow-x: hidden; 
  overflow-y: hidden;
}

.marvel { 
  border: solid 1px #fff;
  animation: leaves 7s ease-in-out forwards;
  -webkit-animation: leaves 7s ease-in-out forwards;
  margin: auto auto;
  text-align: center;
  color: #fff; 
  width: 650px; 
  margin-top: 50px;  
  font-size: 8em; 
  line-height: 1;
  font-family: 'Fjalla One', sans-serif;
  margin-top: 35px;
  padding: 15px 10px 0 10px;
}

.match { 
  font-family: "Orbitron", sans-serif;
  animation: leaves 7s ease-in-out forwards;
  -webkit-animation: leaves 7s ease-in-out forwards;
  font-weight: bolder;
  color: #fff;
  letter-spacing: 67px;
  text-align: center;
  position: relative;
  margin: 15px 15px 5px 85px;
}

body {     
  background: #5d0000;
}

@media (max-width: 320px) {
  .marvel {
    padding: 10px 4px 10px 0;
    width: 280px;
    font-size: 5em;
  }

  .match {
    letter-spacing: 45px;
  }
}