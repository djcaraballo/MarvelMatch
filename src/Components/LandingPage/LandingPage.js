import React from 'react'
import { NavLink } from 'react-router-dom'
import './LandingPage.css'

const LandingPage = () => {
  return (
    <div className="wrapper">
      <div className="logo-container">
        <h1 className="marvel">MARVEL</h1>
        <h2 className="match">MATCH</h2>
      </div>
      <img className="gif" src="../../assets/comic-flip.gif"/>
      <div className="links-container">
        <NavLink 
          to='/UserLoginForm'
          className='landing-nav'>
          <h3>login</h3>
        </NavLink>
        <p><span className="line-span"></span>OR<span className="line-span"></span></p>
        <NavLink 
          to='/NewHeroSignupForm'
          className='landing-nav'>
          <h3>signup</h3>
        </NavLink>
      </div>
    </div>
  )
}

export default LandingPage