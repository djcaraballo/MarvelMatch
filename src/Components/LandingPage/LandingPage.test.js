import React from 'react'
import LandingPage from './LandingPage'
import { shallow } from 'enzyme'

describe('LandingPage Component', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<LandingPage />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})