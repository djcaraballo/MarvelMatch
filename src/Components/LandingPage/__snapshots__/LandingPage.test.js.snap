// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage Component should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LandingPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="marvel"
        >
          MARVEL
        </h1>,
        <h2
          className="match"
        >
          MATCH
        </h2>,
        <img
          className="gif"
          src="../../assets/comic-flip.gif"
        />,
      ],
      "className": "logo-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "MARVEL",
          "className": "marvel",
        },
        "ref": null,
        "rendered": "MARVEL",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "MATCH",
          "className": "match",
        },
        "ref": null,
        "rendered": "MATCH",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "gif",
          "src": "../../assets/comic-flip.gif",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="marvel"
          >
            MARVEL
          </h1>,
          <h2
            className="match"
          >
            MATCH
          </h2>,
          <img
            className="gif"
            src="../../assets/comic-flip.gif"
          />,
        ],
        "className": "logo-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "MARVEL",
            "className": "marvel",
          },
          "ref": null,
          "rendered": "MARVEL",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "MATCH",
            "className": "match",
          },
          "ref": null,
          "rendered": "MATCH",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "gif",
            "src": "../../assets/comic-flip.gif",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
